### DISCUSSION TITLE
Unable to type in anything for newtable

### DISCUSSION THREAD

**Discussion by TLKG** [dbeaver/dbeaver#8853] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version, Win 10.0.18363, Pro
- DBeaver version = 7.0.3202004191823
- Java version, not aware of any as I'm new  
- Additional extensions, not aware of any as I'm new 

#### Connection specification:
- Database name and version = Postgres v12
- Driver name, see below
- Do you use tunnels or proxies (SSH, SOCKS, etc)? not aware of any as I'm new 

#### No error is given.
![image](https://user-images.githubusercontent.com/22269729/83364639-7cd46d00-a357-11ea-8b35-299248676a36.png)
![image](https://user-images.githubusercontent.com/22269729/83364745-59f68880-a358-11ea-95ec-82035725bbb3.png)

**Comment by uslss** [comment]
hi @TLKG ,
you cannot type anything here. to create, e.g., table, you could choose Create New Table in context menu
![image](https://user-images.githubusercontent.com/31996417/83426233-859c7000-a437-11ea-919e-f1561573a507.png)

**Comment by TLKG** [comment]
uslss,
Your screen is BEFORE "Create New Table". 
If you look at my screenshot in detail, I was already inside of Create New Table.

On June 1, 2020 at 8:42 AM, uslss <notifications@github.com> wrote:






hi @TLKG ,
you cannot type anything here. to create, e.g., table, you could choose Create New Table in context menu

**Comment by TLKG** [comment, marked as answer]
Actually I found a way, r-click inside of my newtable screen's Columns area, choose "Create New Column". But this is not intuitive and cumbersome.
![image](https://user-images.githubusercontent.com/22269729/83836902-c6032480-a6a9-11ea-8113-6eb64756475b.png)

**Comment by uslss** [comment]
Oh, shame on me. Seems that issue is solved. If you have any idea how we can improve this, feel free to create feature request or enhancement.
Best regards,
